plugin/src/main/scala/chisel3/internal/plugin/ChiselPlugin.scala

```diff
@@ -8,10 +8,15 @@ import nsc.plugins.{Plugin, PluginComponent}
 import scala.reflect.internal.util.NoPosition
 import scala.collection.mutable
 
-private[plugin] case class ChiselPluginArguments(val skipFiles: mutable.HashSet[String] = mutable.HashSet.empty) {
+private[plugin] case class ChiselPluginArguments(
+  val skipFiles:                mutable.HashSet[String] = mutable.HashSet.empty,
+  var buildElementsAccessor:    Boolean = true,
+  var pluginDebugBundlePattern: String = "" // For example try .*Bpip.*"
```

Comment:
There doesn't seem to be a way to set this via a scalacOption

Reply:
I'm removing this, my docs say to just create this in the code during a debugging session